cluster-autoscaler/cloudprovider/azure/azure_cloud_provider.go

```diff
@@ -200,7 +202,7 @@ func (scaleSet *ScaleSet) Belongs(node *apiv1.Node) (bool, error) {
 
 // DeleteNodes deletes the nodes from the group.
 func (scaleSet *ScaleSet) DeleteNodes(nodes []*apiv1.Node) error {
-	fmt.Printf("Delete nodes requested: %v\n", nodes)
+	glog.Infof("Delete nodes requested: %v\n", nodes)
```

Comment:
There is rarely a need to print the whole node object. However information about the node deletion might be quite useful (it is logged in other places too anyway). So either V(something) + name or remove the line.